### DISCUSSION TITLE
How to handle click table cells (not row)?

### DISCUSSION THREAD

**Discussion by zliucd** [vuetifyjs/vuetify#14944] [General] [answered]
Hi,

I want to handle  click events on **table cells**(of a certain column) using a function. ```v-data-table``` provides click handling for row, but how to handle cell? 

Thank you.

**Comment by nekosaur** [comment, marked as answer]
This is one possible solution https://codepen.io/nekosaur/pen/NWXMJBy?editors=101

**Reply by zliucd** [reply]
Thanks, it works.
